apps/server/src/modules/sharing/service/share-token.service.ts

```diff
@@ -11,8 +11,7 @@ export class ShareTokenService {
 		payload: ShareTokenPayload,
 		options?: { context?: ShareTokenContext; expiresAt?: Date }
 	): Promise<ShareTokenDO> {
-		const token = this.tokenGenerator.generateShareToken();
-
+		const token: string = this.tokenGenerator.generateShareToken();
```

Comment:
Not needed anymore... - it "solved" an eslint error... but later on I realized it was caused by tokenGenerator not working correctly, because of moved DomainObject-file.
Fixed.